server_tests/test/clientTests.js

```diff
@@ -787,89 +789,151 @@ describe('RUST API TEST SUITE', function () {
     if (!isOSC2) {
       return this.skip()
     }
+    var expectedImageCount
+    var expectedVideoCount
+    var totalEntryCount
 
     before(function () {
       // delete exisiting files on SD card, if any
+      return Utility.deleteAllImages() //should have a new utility deleteAll() or something
```

Comment:
Perhaps rather than deleting all images, we just count the current amount and then count the increment? It just seems like perhaps people may want to run this without expecting their card to be wiped.